### DISCUSSION TITLE
Segurança do app

### DISCUSSION THREAD

**Discussion by serpa2022** [emavgl/oinkoin#152] [Ideas]
Primeiramente, agradeço pelo esforço grande em manter um aplicativo simples, mas bem interessante! Com ele, estou organizando minhas finanças! Agradeço de coração! Aí vai uma ideia: poderia ter uma função pra proteger o app com uma senha ou impressão digital, para que, em caso de roubo do celular, tudo fique escondido. Obrigado!

**Comment by emavgl** [comment]
Hi! There is an open issue for that https://github.com/emavgl/oinkoin/issues/14

If the phone gets stolen. Don't you have already a PIN / Biometrics on your lockscreen?

Do you think it really brings some benefit locking the app as well?

Thanks for the feedback. I am happy you like the app as well :)

**Comment by emavgl** [comment]
Done https://github.com/emavgl/oinkoin/releases/tag/1.0.73 - I am closing the issue but feel free to comment.
